microsoft-edge/web-platform/os-regional-settings.md

```diff
@@ -0,0 +1,88 @@
+
+## Three settings
+Customers have three settings they can chose from:
+
+* Never 
+* Limited (default) 
+* Always (not recommended). 
+
+To minimize the risks noted previously, customers can disable the sharing of the OS regional information by choosing “never”. This does not prevent websites from reading region information, but does block any specific OS regional customizations that the user has made from being exposed to website content. Instead, the browser’s display language is used to derive a default region. 
+
+When choosing “Always” the customer is exposed to the full set of risks noted earlier: the potential to be easily signaled out for fingerprinting, and also to potentially see website language-processing problems and inconsistencies. However, the user’s OS preferences will be made available to the website without any limitations or restrictions to be sure their preferences are respected. 
+
+The default recommended value is “limited.” With limited, the above risks are mitigated by ensuring that “incompatible” combinations of language and region are not revealed to websites. The “limited” value ensures that only “valid” combinations of the user’s current browser display language preference and a given region value are used. This ensures that websites won’t see a “regional code” anomaly (though it could still look unusual per geographic inference), and also ensures that for language processing, websites will at least provide a consistent language experience (not mixing languages in weird combinations). The “limited” setting compares the language part of the regional code that is derived from the browser’s display language, and the regional part coming from the OS setting, and if the language parts match and if the resulting regional code is a valid combination, will pass the result through to the website. If the language parts don’t match or if the resultant combination isn’t valid, then the OS region preference is silently ignored and the feature acts as if “never” was selected (the regional information falls back to being derived from the browser’s display language). 
```

Comment:
i'll probably remove some quotes, and maybe wrap the 3 settings names in backticks